src/util/util_env.cpp

```diff
@@ -35,6 +35,13 @@ namespace dxvk::env {
 #endif
   }
 
+  void setEnvVar(const char* name, const char* value) {
+#ifdef _WIN32
+      ::SetEnvironmentVariableA(name, value);
+#else
+      std::setenv(name, value);
```

Comment:
This doesn't exist in the std namespace and has an extra parameter.

I'd be fine with printing a warning and not supporting this in native builds at all for the time being; apps using dxvk-native shouldn't be messing around with the Vulkan environment in the first place and we can still add it later if it does turn out to be needed.

Otherwise, we'd need to explicitly include posix headers here and also make sure it behaves the same as the Windows path (mostly relevant for handling the `value == nullptr` case, but also unicode considerations).